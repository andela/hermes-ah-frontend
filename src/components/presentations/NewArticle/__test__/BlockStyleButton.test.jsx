import React from 'react';
import { shallow } from 'enzyme';
import EditorButton from '../BlockStyles/BlockStyleButton';

const NewArticleButtonComponent = (
  <EditorButton
    // eslint-disable-next-line react/style-prop-object
    style=""
    onToggle={jest.fn()}
    label={jest.fn()}
    active={jest.fn()}
  />
);

describe('Block style button component', () => {
  it('should match snapshot', () => {
    const wrap = NewArticleButtonComponent;
    expect(wrap).toMatchSnapshot();
  });

  it('should render without crashing', () => {
    const wrapper = shallow(NewArticleButtonComponent);
    expect(wrapper.find('button'));
    expect(wrapper.find('form'));
  });
});
