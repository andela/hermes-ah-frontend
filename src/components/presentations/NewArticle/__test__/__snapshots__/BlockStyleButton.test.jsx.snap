// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Block style button component should match snapshot 1`] = `
<BlockStyleButton
  active={[MockFunction]}
  label={[MockFunction]}
  onToggle={[MockFunction]}
  style=""
/>
`;
